content/id/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -23,13 +23,13 @@
     <div class="row">
       <div class="col-md-9">
         <h2>Panduan Dasar Kubernetes</h2>
-        <p>Tutorial ini menyediakan panduan dasar mekanisme orkestrasi kluster Kubernetes. Setiap modul memliki beberapa informasi mengenai latar belakang bagi konsep mendasar dan <i>feature</i> Kubernetes, termasuk mode interaktif yang dapat digunakan sebagai metode pembelajaran <i>online</i>. Mode tutorial interaktif ini memberikan kesempatan pengguna untuk melakukan manajemen kluster sederhana beserta aplikasi dalam kontainer yang kamu miliki.</p>
+        <p>Tutorial ini menyediakan panduan dasar mekanisme orkestrasi kluster Kubernetes. Setiap modul memliki beberapa informasi mengenai latar belakang bagi konsep mendasar dan <i>feature</i> Kubernetes, termasuk mode interaktif yang dapat digunakan sebagai metode pembelajaran <i>online</i>. Mode tutorial interaktif ini memberikan kesempatan pengguna untuk melakukan manajemen kluster sederhana beserta aplikasi dalam container yang kamu miliki.</p>
         <p>Dengan menggunakan mode tutorial interaktif ini, pengguna diharapkan dapat memahami:</p>
         <ul>
-        <li><i>Deploy</i>i> sebuah aplikasi yang sudah dikontainerisasi pada kluster</li>
+        <li><i>Deploy</i>i> sebuah aplikasi yang sudah di-container-isasi pada kluster</li>
         <li>Melakukan <i>scale</i> <i>deployment</i></li>
-        <li>Meng-update aplikasi yang sudah dikontainerisasi dengan menggunakan versi aplikasi terbaru</li>
-        <li>Men-debug aplikasi yang sudah dikontainerisasi</li>
+        <li>Meng-update aplikasi yang sudah di-container-isasi dengan menggunakan versi aplikasi terbaru</li>
```

Comment:
There are many containerization translation that need to standardize whether to be di-container-isasi or dicontainer-isasi or even other translation.

Reply:
@girikuncoro @phanama any idea on this? cc @wahyuoi

Reply:
https://kbbi.kemdikbud.go.id/entri/kontainerisasi

 i think, we can use `kontainerisasi`

Reply:
@wahyuoi nice find! `kontainerisasi` it is then